website/components/typeItAnimation.js

```diff
@@ -0,0 +1,9 @@
+import TypeIt from "typeit-react"
+
+export default function TypeItAnimation() {
+    return (
+        <div>
+             <TypeIt>nfddfdfdfdfdffd</TypeIt>
```

Comment:
Is this a correct content? 🤔

Reply:
Im gonna finish this component today

Reply:
Cool. It's your call then to merge this PR.